scripts/build-k8s-on-amd64/build-k8s.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+echo "OUTPUT=$OUTPUT"
+
+# Build etcd
+curl -sSL https://github.com/coreos/etcd/archive/$ETCD_VERSION.tar.gz | tar -C $ETCD_DIR -xz --strip-components=1 \
+    && cd $ETCD_DIR \
+    && ./build \
+    && cp bin/* $OUT_DIR
+
+# Build flannel
+curl -sSL https://github.com/coreos/flannel/archive/$FLANNEL_VERSION.tar.gz | tar -C $FLANNEL_DIR -xz --strip-components=1 \
+    && cd $FLANNEL_DIR  \
+    && CGO_ENABLED=1 ./build \
+    && cp bin/* $OUT_DIR
+
+# Build kubernetes
+curl -sSL https://github.com/kubernetes/kubernetes/archive/$K8S_VERSION.tar.gz | tar -C $K8S_DIR -xz --strip-components=1 \
+    && cd $K8S_DIR \
+    && CGO_ENABLED=1 ./hack/build-go.sh cmd/hyperkube cmd/kubectl \
+    && cp $OUTPUT/* $OUT_DIR
+
+# Build the pause image
```

Comment:
This isn't required anymore :)
The official (built with C) image gcr.io/google_containers/pause-arm:3.0 image is pushed, so we may drop this now